### ISSUE TITLE
[1.14.4] Loading error (evtl. config related)

### ISSUE THREAD

**Issue by ZephyrWish** [MC-U-Team/U-Team-Core#21]
[crash-2019-08-12_15.36.13-client.txt](https://github.com/MC-U-Team/U-Team-Core/files/3491388/crash-2019-08-12_15.36.13-client.txt)

Only Forge and UCore are installed. Minecraft opens fine but near the end of the loading bar it crashes.

**Comment by MrTroble** [comment]
Hallo and thanks for your report.
We will look into it asap.

Cheers
~MrTroble

**Comment by HyCraftHD** [comment]
Cannot reproduce. Is loading fine with these mods.
![grafik](https://user-images.githubusercontent.com/7681220/62854583-04c98780-bcf0-11e9-8498-1ea649a2959a.png)

The issues seems like a configuration file is broken, or your pc ran out of ram etc.
Can you try to delete your config folder and try to start again?

**Comment by ZephyrWish** [comment]
You are right the config file is broken. I looked into the config folder and the teamcore.toml looked like this
![Untitled1](https://user-images.githubusercontent.com/54019774/62860384-9eb22400-bd32-11e9-8b51-2d2be8956acd.png)

I deleted it, reopened MC and it works perfectly now
![Untitled2](https://user-images.githubusercontent.com/54019774/62860476-dcaf4800-bd32-11e9-9f43-ba1281005e5b.png)

I dont know what caused the file to become corrupt? It just broke one day when i upgraded forge versions. Thanks for helping me fix this really simple issue lmao <3
